lingua_franca/format.py

```diff
@@ -370,6 +374,39 @@ def nice_date_time(dt, lang='', now=None, use_24hour=False,
                                              use_ampm)
 
 
+@localized_function(run_own_code_on=[FunctionNotLocalizedError])
+def nice_day(dt, date_format='MDY', include_month=True, lang=""):
```

Comment:
perhaps `date_format` should be an enum since there are only 3(?) valid options right now (MDY, DMY, YMD).

Reply:
can't hurt, but note this comes from mycroft.conf and the valid options are basically defined there, so let's define what the enum should contain clearly